src/sections/Community/Handbook/contributing.js

```diff
@@ -146,6 +150,30 @@ const contributingGuide = () => {
                 </span>
               </li>
             </ul>
+            <a id="Create a pull request">  <h3>Create a pull request (PR)</h3> </a>
+            <ul>
+              <li>
+                <span>
+                  Head over to your forked repository on GitHub and you'll see a Compare & pull request button. Click on that button.
+                </span>
+                <div>
+                  <br />
+                  <StaticImage src={ compareAndPR }  alt="Compare and pull request" />
+                </div>
+              </li>
+            </ul>
+            <ul>
+              <li>
+                <span>
+                  Once you click on the button, you'll be taken to the Pull Request page. Here, you can add a title and description to your pull request that explains your contribution. Once you're done, click on the Create pull request button.
+                </span>
+                <div>
```

Comment:
A note should also be added in here about check if the right branches are being compared for which the PR is to be made. 

PS:- As in some cases it doesn't , and the PR results with the users forked repo master branch itself.

Reply:
Thankyou I'll add this as well